### PR TITLE
StringManipulator_GKundu

### PR DESCRIPTION
**Pull request by GKundu**

### CODE REVIEW

File: src/hw1/StringManipulator.java [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
+        
+         /*Write a program that asks the user to enter the name of his or her favorite city. 
          Use a String variable to store the input. The program should display the following:
          The number of characters in the city name
          The name of the city in all uppercase letters
          The name of the city in all lowercase letters
-         The first character in the name of the city
-         */
+         The first character in the name of the city*/
+         System.out.println("Please enter your favorite city: ");
```
Comment (khatchad) [inline src/hw1/StringManipulator.java:24]: Please format your source code prior to submission.

File: src/hw1/StringManipulator.java [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
+         Scanner sc = new Scanner(System.in);
+
+        String in = sc.nextLine();
+        String Upper = in.toUpperCase();
```
Comment (khatchad) [inline src/hw1/StringManipulator.java:28]: Variable names should start with a lowercase letter, e.g., `upper`.

File: src/hw1/StringManipulator.java [hunk 3]
```diff
@@ -1,24 +1,38 @@
...
+        String Lower = in.toLowerCase();
+        int numberofletter = in.length();
```
Comment (khatchad) [inline src/hw1/StringManipulator.java:30]: In camel case: `numberOfLetters`. Actually, you probably want `numberOfCharacters` as punctuation counts.

**Comment by GKundu** [comment]
Thanks For the Feedback

**Pull request closed without merging**